src/services/healthcareProfessionalService.ts

```diff
@@ -306,6 +306,7 @@ function validateNames(
     names: gqlTypes.LocalizedNameInput[] | undefined | null,
     validationResults: Result<unknown>
 ): void {
+    console.log("Names:", names)
```

Comment:
I'm assuming this was left in by mistake.

Reply:
Yes, I will make sure everything is cleaned up if I make my final PR 😊